akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -163,14 +180,16 @@ final class ORMultiMap[A, B] private[akka] (
    * Convenience for remove. Requires an implicit Cluster.
    * @see [[#remove]]
    */
-  def -(key: A)(implicit node: Cluster): ORMultiMap[A, B] =
-    remove(node, key)
+  @deprecated("Use `remove` that takes a `SelfUniqueAddress` parameter instead.", since = "2.5.19")
+  def -(key: A)(implicit node: Cluster): ORMultiMap[A, B] = remove(node.selfUniqueAddress, key)
 
   /**
    * Remove an entire set associated with the key.
    */
-  def remove(node: Cluster, key: A): ORMultiMap[A, B] =
-    remove(node.selfUniqueAddress, key)
+  def remove(node: SelfUniqueAddress, key: A): ORMultiMap[A, B] = remove(node.uniqueAddress, key)
```

Comment:
also:
```
def remove(key: A)(implicit node: SelfUniqueAddress): ORMultiMap[A, B]
```

for Scala API as replacement for `-`

Reply:
I believe I tried that and it conflicted with the old but I’ll try again now.